adapt-and-extend/theming/customize-barceloneta.rst

```diff
@@ -0,0 +1,181 @@
+=======================================
+Customize the Barceloneta default theme
+=======================================
+
+TODO
+
+
+Heading
+-------
+
+TODO
+
+
+Subheading
+++++++++++
+
+TODO
+
+
+Subheading
+++++++++++
+
+TODO
+
+
+Heading
+-------
+
+TODO
+
+
+Introduction
+------------
+
+Target group
+++++++++++++
+
+A Systems integrator with average knowledge about Plone. Aims to theme a Plone site. 
+
+
+Aim
++++
+
+In this tutorial you will learn how to build a custom theme for your website by changing several elements of Plone. You will learn where to find and how to customise different building blocks in Plone while keeping the basic concepts and structure intact. 
+
+Demarcation
++++++++++++
+
+This is not about creating a standalone theme which behaves independently from Plone UI concepts. If you plan to do this, you should read the next scenario.
+
+Requirements
+++++++++++++
+
+We assume that you want to keep your customisations in filesystem code. You are  familiar with how to create and register an egg to accommodate the customisations. You can e.g. create such an egg by using mr.bob and following the mr.bob documentation on pypi <https://github.com/plone/bobtemplates.plone>`_ and picking the theme option.
+
+
+
+Getting started with the Basics
+-------------------------------
+
+New logo
+++++++++
+
+Probably the very first thing you want to change in a new site is the logo and the favicon. Let’s start with the logo.
+
+
+First we override the existing logo using `z3c.jbot <https://pypi.python.org/pypi/z3c.jbot>`_. Hierfür gibt es in unserem neu erstellten Paket bereits den Ordner ``overrides`` in ``your.egg/src/your/egg/browser/``.
+
+::
+
+    $ cd your.egg/src/your/egg/browser/overrides/
+    $ cp fromplonepositionoflogo .
+    $ mv logo.png Products.CMFPlone.skins.plone_images.logo.png
+
+We assume you 
+Anschließend kann die Instanz neu gestartet werden, eine Plone-Site erstellt und ``vs.base`` aktiviert werden.
```

Comment:
having German amidst the docs is probably confusing for the non-German audience...